Gemfile

```diff
@@ -1,4 +1,4 @@
-source :rubygems
+source 'https://rubygems.org'
 
 gem 'activesupport', '>= 2.3.11'
```

Comment:
Since it is a git link and set to "master" branch, I am expecting it to
pull the code every time... but I'm not 100% sure.